crates/revm/src/db/states/bundle_state.rs

```diff
@@ -100,9 +100,14 @@ impl BundleBuilder {
     }
 
     /// Collect storage info of BundleState state
-    pub fn state_storage(mut self, address: Address, storage: HashMap<U256, (U256, U256)>) -> Self {
+    pub fn state_storage(
+        mut self,
+        address: Address,
+        storage: &mut HashMap<U256, (U256, U256)>,
```

Comment:
This is not what the requirement was.

It is to add `get_state_storage_mut(` function

Reply:
While we are at it, it would be good to add mut getters for all fields

Reply:
my bad, i'm going to fix it 💪